### DISCUSSION TITLE
📣 NeoVim users, please upgrade to version 1.3.0

### DISCUSSION THREAD

**Discussion by davecheney** [community/community#16423] [Copilot Conversations]
NeoVim pilots :wave: older versions of the copilot.vim extension are no longer supported. Please upgrade to version 1.3.0 of your copilot.vim extension.

**Comment by andyengit** [comment]
ty

**Reply by 13223926092** [reply]
I am a college student majoring in computer science from China and am very interested in GitHub Copilot. It is a pity that I am not on the list. Can you help me?

**Reply by davecheney** [reply]
Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

**Comment by ghost** [comment]
![20210108_230829](https://user-images.githubusercontent.com/105374724/168571476-28f84303-af47-4587-a2c6-a768b2465818.png)

**Comment by ghost** [comment]
![43705](https://user-images.githubusercontent.com/105374724/168571739-d68bf871-624f-4ee0-8fa6-2e7f3ecb5dd4.JPEG)

**Comment by dkbast** [comment]
if possible it would be great to include these service announcements in the plugin.

I would suggest checking the local version against the oldest supported version (could even just be a raw file in the plugins repo) and  displaying "Please update, version no longer supported" instead of "Enabled and online" 

Browniepoints for also pointing out if there is any update available :) - anyways thanks for this great plugin!

**Comment by wyattverdrell173-blip666** [comment]
100
